docs/documentation/server_admin/topics/authentication/flows.adoc

```diff
@@ -403,6 +403,25 @@ Note when the login request initiates a request with the `claims` parameter requ
 one of the specified levels. If it is not able to return one of the specified levels (For example if the requested level is unknown or bigger than configured conditions
 in the authentication flow), then {project_name} will throw an error.
 
+[[_registration-rc-client-flows]]
+==== Registration or Reset credentials requested by client
+
+Usually when user is redirected to the {project_name} from client application, the `browser` flow is triggered. This flow may allow user to <<con-user-registration_{context}, register>> in case
+that realm registration is enabled and user clicks to the link `Register` on the login screen. Also in case that <<enabling-forgot-password, Forget password>> is enabled for the realm, user can
+click on the link `Forget password` on the login screen, which will trigger `Reset credentials` flow where user can reset his credentials after confirmation of his email address.
+
+Sometimes it can be useful for client application to directly redirect user to the *Registration* screen or to *Reset credentials* flow. This will have exactly same behaviour like when manually
```

Comment:
```suggestion
Sometimes it can be useful for the client application to directly redirect the user to the *Registration* screen or to the *Reset credentials* flow. This resulting action will match the action of when the 
```